src/locales/en/en.json

```diff
@@ -875,6 +875,10 @@
               "message": "You are currently connected. To connect with a new wallet, you will be disconnected from your current selection. Would you like to disconnect and continue?",
               "confirm": "Continue",
               "cancel": "Cancel"
+            },
+            "connectionbrokenalert": {
```

Comment:
Super tiny thing but this should be `connectwallet.connectionbrokenalert` instead of `connectwallet.connectwalletmodal.connectionbrokenalert` as its not part of the modal